tests/migrations/test_executor.py

```diff
@@ -649,6 +653,24 @@ def test_migrate_marks_replacement_applied_even_if_it_did_nothing(self):
             recorder.applied_migrations(),
         )
 
+    @skipIfDBFeature('can_rollback_ddl')
```

Comment:
You want to run this test only if the backend has `can_rollback_ddl` instead. The reason for this is that when the feature is false the operation and the record won't be performed in a transaction and your test will systematically pass.

https://github.com/django/django/blob/58d1e9aa8ab505912389e7cd019a6f21785ad4bf/django/db/backends/base/schema.py#L86-L100

That should be `@skipUnlessDbFeature('can_rollback_ddl') instead.

Reply:
I've tried to do `from django.test import skipUnlessDBFeature` and `from django.test.testcases import skipUnlessDBFeature` but both of them are failing. I'm trying to look into this one.